scripts/scripts.js

```diff
@@ -140,12 +140,21 @@ function decorateQuote() {
 
 const { loadArea, setConfig } = await import(`${miloLibs}/utils/utils.js`);
 
+function overrideMiloToc() {
+  const tocs = document.querySelectorAll('.table-of-contents');
+  tocs.forEach((toc) => {
+    toc.classList.remove('table-of-contents');
+    toc.classList.add('table-of-contents-override');
```

Comment:
Instead of using `override` what about something like `blog` or `simple`?
`table-of-contents-blog`
`table-of-contents-simple`

@chrischrischris @auniverseaway Can you think of a better substitute?

Reply:
Lets go with `blog-table-of-contents`

Reply:
Agreed with prepending the project first.  If you wanted it shorter I'd also be ok with `blog-toc`

Reply:
I renamed the block `blog-table-of-contents`: see https://github.com/adobecom/blog/pull/15/commits/c7366cfd7bdb67ff376dd7a1e5327438b85b6616